core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -3151,9 +3152,31 @@ private[spark] object Utils
   }
 }
 
-private[util] object CallerContext extends Logging {
-  val callerContextEnabled: Boolean =
-    SparkHadoopUtil.get.conf.getBoolean("hadoop.caller.context.enabled", false)
+private[spark] object CallerContext extends Logging {
+  var callerContextEnabled: Boolean = SparkHadoopUtil.get.conf.getBoolean(
+    HADOOP_CALLER_CONTEXT_ENABLED_KEY, HADOOP_CALLER_CONTEXT_ENABLED_DEFAULT)
```

Comment:
If you don't mind, please revert this change. :)

Reply:
No problem, I can definitely do that. :)

Can you please help me understand the reasoning though? Does the Spark codebase prefer not to reference Hadoop's configuration constants?

Reply:
Yes, right. We prefer to avoid those compilation dependency. Not only for Hadoop, but also Hive, too.

Reply:
Thank you, and I will keep it in mind in future patches.